app/src/main/java/com/stevesoltys/seedvault/settings/SettingsManager.kt

```diff
@@ -134,13 +146,24 @@ class SettingsManager(private val context: Context) {
     fun canDoBackupNow(): Boolean {
         val storage = getStorage() ?: return false
         val systemContext = context.getStorageContext { storage.isUsb }
-        return !storage.isUnavailableUsb(systemContext) && !storage.isUnavailableNetwork(context)
+        return !storage.isUnavailableUsb(systemContext) &&
+            !storage.isUnavailableNetwork(context, useMeteredNetwork)
     }
 
     fun backupApks(): Boolean {
         return prefs.getBoolean(PREF_KEY_BACKUP_APK, true)
     }
 
+    val backupFrequencyInMillis: Long
```

Comment:
I assume we plan on exposing this to the user?

Reply:
It is already exposed and configurable via `SchedulingFragment`.